worlds/sc2/locations.py

```diff
@@ -733,6 +733,8 @@ def get_locations(world: Optional['SC2World']) -> Tuple[LocationData, ...]:
         make_location_data(SC2Mission.HARVEST_OF_SCREAMS.mission_name, "Nafash", SC2HOTS_LOC_ID_OFFSET + 408, LocationType.EXTRA,
                      lambda state: logic.zerg_common_unit(state)
                                    and logic.zerg_basic_anti_air(state)),
+        make_location_data(SC2Mission.HARVEST_OF_SCREAMS.mission_name, "20 Unfrozen Structures", SC2HOTS_LOC_ID_OFFSET + 409, LocationType.CHALLENGE,
+                     lambda state: logic.zerg_common_unit(state)),

```

Comment:
And Basic AA or competent AA. Scouts are nearly everywhere

Reply:
Good call. Added `zerg_basic_anti_air`. Scouts aren't a huge threat so I think basic should be sufficient.